jcl/src/java.base/share/classes/java/lang/StackWalker.java

```diff
@@ -391,8 +391,17 @@ public StackTraceElement toStackTraceElement() {
 					moduleVersion = versionInfo.get().toString();
 				}
 			}
-			return new StackTraceElement(classLoaderName, moduleName, moduleVersion, className, methodName, fileName,
+
+			StackTraceElement element = new StackTraceElement(classLoaderName, moduleName, moduleVersion, className, methodName, fileName,
 					lineNumber);
+
+			/**
+			 * Disable including classloader name and module version in stack trace output
+			 * until StackWalker StackTraceElement include info flags can be set properly
```

Comment:
Please add a period to end the sentance. Also create a new issue for the future change and refer to it.

Reply:
Added period; now referring to https://github.com/eclipse/openj9/issues/11774